lib/widgets/receivable_btc_box.dart

```diff
@@ -57,11 +57,9 @@ class ReceivableBTCBoxState extends State<ReceivableBTCBox> {
                 : AutoSizeText(
                     widget.receiveLabel ??
                         texts.invoice_receive_label(
-                          currencyState.bitcoinCurrency.format(
-                            (isChannelOpeningAvailable)
-                                ? accountState.maxInboundLiquidity
-                                : accountState.maxAllowedToReceive,
-                          ),
+                          currencyState.bitcoinCurrency.format((!isChannelOpeningAvailable)
+                              ? accountState.maxInboundLiquidity
+                              : accountState.maxAllowedToReceive),
```

Comment:
NIT Suggestion:

Swap conditions for readability.
```suggestion
                          currencyState.bitcoinCurrency.format((isChannelOpeningAvailable)
                              ? accountState.maxAllowedToReceive
                              : accountState.maxInboundLiquidity),
```

Reply:
Done